src/components/header/SubHeader.astro

```diff
@@ -47,6 +56,22 @@ const shaking = version === 'dev' ? '3deg' : '1deg';
                 ''
             )
         }
+        {
+            !isDsl2 ? (
+                <div class="alert alert-warning p-1 p-md-3" role="alert">
+                    <span>
+                        This pipeline uses DSL1. From Nextflow version <code>22.03.0-edge</code> DSL2 is the default DSL
+                        version.{' '}
+                        <a href="https://www.nextflow.io/docs/edge/dsl1.html" target="_blank">
+                            Learn more
+                        </a>
+                        .
```

Comment:
```suggestion
                        This pipeline uses DSL1. It will not work with Nextflow version after <code>22.03.0-edge</code>  {' '}
                        <a href="https://www.nextflow.io/docs/edge/dsl1.html" target="_blank">
                            Learn more
                        </a>.
```